src/htmlContent/themes-settings.html

```diff
@@ -30,20 +30,20 @@ <h1 class="dialog-title">{{Strings.THEMES_SETTINGS}}</h1>
             <div class="control-group">
                 <label class="control-label">{{Strings.FONT_SIZE}}:</label>
                 <div class="controls">
-                    <input type="text" data-target="fontSize" value="{{settings.fontSize}}">
+                    <input type="text" id="font-size-input" data-target="fontSize" value="{{settings.fontSize}}">
                 </div>
             </div>
 
             <div class="control-group">
                 <label class="control-label">{{Strings.FONT_FAMILY}}:</label>
                 <div class="controls">
-                    <input type="text" data-target="fontFamily" value="{{settings.fontFamily}}">
+                    <input type="text" id="font-family-input" data-target="fontFamily" value="{{settings.fontFamily}}">
```

Comment:
don't need to introduce id's here, you can use `.on("input", "[data-target=fontFamily]", function () {`
https://www.w3schools.com/jquery/sel_attribute_equal_value.asp